### PR TITLE
Web - Adding MSI support

### PR DESCRIPTION
**Pull request by LukaszStem**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/web/resource-manager/Microsoft.CertificateRegistration/2015-08-01/AppServiceCertificateOrders.json [hunk 1]
```diff
@@ -1354,6 +1354,32 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
+        "identity": {
```
Comment (veronicagg) [inline specification/web/resource-manager/Microsoft.CertificateRegistration/2015-08-01/AppServiceCertificateOrders.json]: @ravbhatnagar should this definition match https://github.com/Azure/azure-rest-api-specs-pr/blob/current/specification/common-types/resource-management/v1/types.json#L200? I see the enum as modelAsString: true here, but false in the "common-types". 
Also, could we match the descriptions?

**Review by veronicagg — CHANGES_REQUESTED** [review]
Are these changes already in the deployed APIs and the PR is correcting the spec? or are these recent updates to the existing APIs?

File: specification/web/resource-manager/Microsoft.CertificateRegistration/2015-08-01/AppServiceCertificateOrders.json (hunk 1 above)
Comment (lmazuel) [reply specification/web/resource-manager/Microsoft.CertificateRegistration/2015-08-01/AppServiceCertificateOrders.json]: @veronicagg we should be consistent with compute:
https://github.com/Azure/azure-rest-api-specs/blob/current/specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json#L4709-L4734

Consistency here will simplify a lot samples at the SDK level

Comment (LukaszStem) [reply specification/web/resource-manager/Microsoft.CertificateRegistration/2015-08-01/AppServiceCertificateOrders.json]: Fixed

**Comment by veronicagg** [comment]
thanks @LukaszStem . @ravbhatnagar any concerns?

**Comment by ravbhatnagar** [comment]
@LukaszStem - Looks fine but shouldnt this support be enabled in a newer API-version?

**Comment by LukaszStem** [comment]
This is additive so all previous api-versions support it. It is backward compatible.

**Comment by ravbhatnagar** [comment]
Looks good! signing off

**Comment by LukaszStem** [comment]
@veronicagg these changes are already in production. LGTM

**Review by veronicagg — APPROVED** [review]

**Pull request merged**